launch.go

```diff
@@ -171,6 +194,24 @@ func launch(api screwdriver.API, buildID int, rootDir, emitterPath string) error
 		return fmt.Errorf("fetching Pipeline ID %d: %v", j.PipelineID, err)
 	}
 
+	log.Printf("Fetching Parent Build %d for Meta", b.ParentBuildID)
+	pb, err := api.BuildFromID(b.ParentBuildID)
+	if err != nil {
+		return fmt.Errorf("fetching Parent Build ID %d: %v", ParentBuildID, err)
```

Comment:
It seems that these are lowercase comments and uppercase comments.
Does `log` use uppercase and `error` use lowercase?

fixed to `pb.ParentBuildID`.